test/1_stdlib/PrintDiagnostics.swift

```diff
@@ -2,9 +2,9 @@
 
 var stream = ""
 
-print(3, &stream) // expected-error{{'&' used with non-inout argument of type 'Any'}}
-debugPrint(3, &stream) // expected-error{{'&' used with non-inout argument of type 'Any'}}
+print(3, &stream) // expected-error{{'&' used with non-inout argument of type 'protocol<>'}}
+debugPrint(3, &stream) // expected-error{{'&' used with non-inout argument of type 'protocol<>'}}
```

Comment:
Replacing `Any` with `protocol<>` in the error messages in this PR is a temporary workaround; I plan on making another PR to fix how compositions are printed/demangled once this one is merged. 

There are other errors with printing too, like the message `'(protocol<>).Type' (aka 'protocol<>.Type')` which have appeared and I plan on fixing.